docs/kb/intro-about.md

```diff
@@ -8,12 +8,12 @@ cta:
  title: Getting Started
  description: Get started with VMware Event Broker Appliance and extend your vSphere SDDC in under 60 minutes
  actions:
-    - text: Install the [Appliance with Knative](install-knative) to extend your SDDC with our [community-sourced functions](/examples)
+    - text: Install the [Appliance](install-veba) to extend your SDDC with our [community-sourced functions](/examples)
 ---
 
 # VMware Event Broker Appliance
 
-The [VMware Event Broker Appliance](https://flings.vmware.com/vmware-event-broker-appliance#summary) Fling enables customers to unlock the hidden potential of events in their SDDC to easily create [event-driven automation](https://octo.vmware.com/vsphere-power-event-driven-automation/). The VMware Event Broker Appliance includes support for vCenter Server and VMware Horizon events as well as any valid `CloudEvent` through the native webhook event provider. Easily triggering custom or prebuilt actions to deliver powerful integrations within your datacenter across public cloud has never been more easier before. A detailed list of use cases and possibilities with VMware Event Broker Appliance is available [here](https://vmweventbroker.io)
+The [VMware Event Broker Appliance](https://flings.vmware.com/vmware-event-broker-appliance#summary) Fling enables customers to unlock the hidden potential of events in their SDDC to easily create [event-driven automation](https://octo.vmware.com/vsphere-power-event-driven-automation/). The VMware Event Broker Appliance includes support for vCenter Server and VMware Horizon events as well as any valid `CloudEvent` through the native webhook event provider. Easily triggering custom or prebuilt actions to deliver powerful integrations within your datacenter across public cloud has never been more easier before. A detailed list of use cases and possibilities with VMware Event Broker Appliance is available [here](https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/development/USECASES.md).
```

Comment:
VEBA Fling URL needs to go to via.vmware.com/flings

Reply:
Done